### ISSUE TITLE
Request: Changeable Splash Art Support

### ISSUE THREAD

**Issue by SinsOfSeven** [BuIlDaLiBlE/BetterHI3Launcher#2]
The GLB PC Client for Hi3 is slightly different from the SEA Client, it happens to have a config.ini and a png that controls the display image.
```ini
[General]
cps=
game_install_path=C:/Program Files/Honkai Impact 3rd/Games
game_dynamic_bg_name=210121140632.png
```

Anyways, I really like to edit the Hi3 Launcher images to make the experience more personal, and I also think that it would be cool for SEA players to have that option as well. When I dropped in the BtrHi3 launcher I noticed it ignored the .ini so I don't have my cute splash image. I really like your launcher and this would just be a small improvement. I'd do it myself but... I really don't know where to start.

Just to show why it matters to me.
~~what I really want to ask for is control over all the UI elements, but I could just re-learn how to build my own from the source.~~
Just doing it how the NA client does it should be pretty easy. I think.

![unknown (10)](https://user-images.githubusercontent.com/26401302/107853051-1c943700-6dd1-11eb-8f46-f692e24370c0.png)
![Launcher_images_examples](https://user-images.githubusercontent.com/26401302/107853083-43eb0400-6dd1-11eb-84cf-e7a0ed65c953.png)
Idk if I should source these here, Yikes. 
I'll just call them transformative content. TyVm artists.

**Comment by BuIlDaLiBlE** [comment]
Nice suggestion, I'll look into it sometime.
For the time being you can actually just edit the image that's already in `%userprofile%\AppData\LocalLow\Bp\Better HI3 Launcher` :^)

**Comment by SinsOfSeven** [comment]
Thanks for letting me know it's over in APPDATA.

**Comment by BuIlDaLiBlE** [comment]
Implemented in [v1.2.20210613.0](https://github.com/BuIlDaLiBlE/BetterHI3Launcher/releases/tag/1.2.20210613.0). ~~Forgot to close this lol.~~
